python/gtsam_unstable/gtsam_unstable.tpl

```diff
@@ -9,6 +9,7 @@
 
 #include <pybind11/eigen.h>
 #include <pybind11/stl_bind.h>
+#include <pybind11/stl.h>
```

Comment:
Is it possible to do this without `stl.h`? If not it's good then.

Reply:
We already have `stl.h` in the `gtsam.tpl` so this is fine I guess?